src/hooks/useTransactions.ts

```diff
@@ -14,23 +16,50 @@ interface UseTransactionsPayload {
 
 interface UseTransactionsParams {
   fromDate?: Date;
+  safeConfig: SafeConfig;
```

Comment:
linked to my other comment, here I'd have
```suggestion
  safeConfig?: SafeConfig;
```